clang/include/clang/APINotes/Types.h

```diff
@@ -737,6 +737,9 @@ inline bool operator!=(const TypedefInfo &LHS, const TypedefInfo &RHS) {
   return !(LHS == RHS);
 }
 
+/// The file extension used for the source representation of API notes.
+static const char SOURCE_APINOTES_EXTENSION[] = "apinotes";
```

Comment:
Perhaps add a `constexpr` to this as well?

Reply:
Done